src/models/response/cipherResponse.ts

```diff
@@ -38,7 +39,12 @@ export class CipherResponse extends BaseResponse {
         this.name = this.getResponseProperty('Name');
         this.notes = this.getResponseProperty('Notes');
         this.favorite = this.getResponseProperty('Favorite') || false;
-        this.edit = this.getResponseProperty('Edit') || true;
+        this.edit = !!this.getResponseProperty('Edit');
+        if (this.getResponseProperty('ViewPassword') == null) {
```

Comment:
Given this will be a non null field in the DB, required through the server-side pipeline, would there ever be a case where this would be `null`?

Reply:
I added the check to ensure compatibility with older releases of the server. This will be null should the client connect to an old server which would cause `viewPassword` to be `false`, which is undesired.